extensions/prototype-content-with-issues-table/app.R

```diff
@@ -0,0 +1,197 @@
+library(shiny)
+library(bslib)
+library(DT)
+library(connectapi)
+library(dplyr)
+library(purrr)
+
+# cache data to disk with a refresh every 8h
+shinyOptions(
+  cache = cachem::cache_disk("./app_cache/cache/", max_age = 60 * 60 * 8)
+)
+
+source("get_usage.R")
+
+# Hacky function to get a list of Content class objects without making a request
+# for each item. These objects differ from the ones created by `content_item()`
+# because they also include the full owner info as returned by `get_content()`.
```

Comment:
This sounds like a feature request for `connectapi` have you raised an issue there?

Reply:
I'll check to make sure @toph-allen doesn't have an issue in the works, as I pulled this helpful function from him during code pairing, and will open one if not.